### DISCUSSION TITLE
Help configuring tampermonkey

### DISCUSSION THREAD

**Discussion by FoulestAtol** [Farigh/pokeclicker-automation#301] [Q&A] [answered]
I have tamper monkey but what do i do now. could someone help please i need it.

**Comment by FoulestAtol** [comment]
im new to git hub so idk what to do

**Comment by Farigh** [comment, marked as answer]
Hi, sorry for the late reply, I did not have much time lately

To add a tampermonkey script, you must click on the add-on icon and create a new script : 
![image](https://github.com/Farigh/pokeclicker-automation/assets/11090416/6f08f936-ce4a-49d2-b7d0-28661d268bc4)

Once done, paste the content of [Automation-UserScript.js](https://github.com/Farigh/pokeclicker-automation/blob/master/Automation-UserScript.js) in the editor that was opened in a new tab : 
![image](https://github.com/Farigh/pokeclicker-automation/assets/11090416/bab3d4c3-25e5-4567-96c6-314d17de5a08)
And save.

Once done, simply go to https://pokeclicker.com and it should load the script automatically
